src/config.js

```diff
@@ -47,7 +47,7 @@ function loadConfig() {
     bitlyToken: {
       doc: `This token is used to access the bitly API`,
       format: String,
-      default: '',
+      default: process.env.NODE_ENV === 'test' ? 'token' : '',
```

Comment:
A string like `'FAKE_TOKEN_FOR_TESTS'` may be a bit more explicit.

Reply:
good idea